### DISCUSSION TITLE
New open, 네 번째 책 선정! - <Clean Code 클린 코드 애자일 소프트웨어 장인 정신 >

### DISCUSSION THREAD

**Discussion by silqwer** [silqwer/book-club#2] [Show and tell]
## 사전 논의

- https://github.com/virnect-corp/BookClub/discussions/96

위의 책 후보들 중에서 어느 걸 먼저 하는지 얘기하다가 차례대로 하자는 의견대로 해서 진행합니다.

1주일 휴식 및 후기 모임을 갖은뒤
읽기 시작: 11월 11일 금요일 부터
첫 리뷰 모임: 11월 25일 금요일 점심

##  Book

[Clean Code 클린 코드 애자일 소프트웨어 장인 정신](http://aladin.kr/p/cGVOm)

![Clean Code 클린 코드 애자일 소프트웨어 장인 정신](https://image.aladin.co.kr/product/3408/36/cover500/8966260950_2.jpg)

## Splint

- 2022-11-11 ~ 2022-11-25: 1장 깨끗한 코드, 2장 의미있는 이름, 3장 함수
- 2022-11-25 ~ 2022-12-09: 4장 주석, 5장 형식 맞추기, 6장 객체와 자료 구조
- 2022-12-09 ~ 2022-12-23: 7장 오류 처리, 8장 경계, 9장 단위 테스트, 10장 클래스
- 2022-12-23 ~ 2023-01-06: 11장 시스템, 12장 창발성, 13장 동시성
- 2023-01-06 ~ 2023-01-20: 14장 점진적인 개선
- 2023-01-20 ~ 2023-02-03: 15장 JUnit 들여다보기, 16장 SerialDate 리팩터링
- 2023-02-03 ~ 2023-02-25: 17장 냄새와 휴리스틱, 부록A 동시성 2

## Rules

- Meet up in office at lunch time Friday end of splint day.
- Create pull request review md file every member one days before meet up day.
- To approve review, write comment to each member until meet up time.

**Comment by jongfeel** [comment]
제 예상에는 12월 23일이 마지막 오프라인 모임이 될 것 같고
그 이후에 2023년 부터는 온라인으로 봐야 할 것 같군요.

각자 사정과 시간을 고려해서 결정하면 좋을 것 같네요.

**Reply by silqwer** [reply]
전 오프라인도 좋아욤 🤠

**Comment by silqwer** [comment]
드디여 대장정의 마무리! 👍

**Reply by jongfeel** [reply]
수고 하셨습니다. 끝이 나서 다행이에요.
